include/PropertiesUtils.h

```diff
@@ -33,6 +33,7 @@ namespace PropertiesUtils {
  * This function doesn't modify the given str.
  */
 std::string LeftTrim(const std::string& str);
+std::string ltrim(const std::string& str);
```

Comment:
Since `ltrim` is more like a helper function, I think it's better to not expose this in the header.